pkg/pillar/Dockerfile.in

```diff
@@ -67,6 +67,7 @@ ADD scripts/device-steps.sh \
     scripts/generate-self-signed.sh \
     scripts/handlezedserverconfig.sh \
     scripts/veth.sh \
+    scripts/monitor-newlogmgr.sh \
```

Comment:
Why is this part of pillar container?

Reply:
IOW, shouldn't we assume that this will go into its separate container along the lines of the current rsyslog container?